### PR TITLE
Improve `wp shell` command description

### PR DESCRIPTION
**Pull request by hearvox**
Related: [/wp-cli/issues/4302](https://github.com/wp-cli/wp-cli/issues/4302)

Improve `wp shell` command description in:
https://github.com/wp-cli/shell-command/blob/master/src/Shell_Command.php
https://github.com/wp-cli/shell-command/blob/master/README.md

Current description: Interactive PHP console.

New description:
Opens an interactive PHP console for running and testing PHP code.

### CODE REVIEW

File: src/Shell_Command.php [hunk 1]
```diff
@@ -3,7 +3,7 @@
 class Shell_Command extends \WP_CLI_Command {
 
 	/**
-	 * Interactive PHP console.
+	 * Opens an interactive PHP console for running and testing PHP code.
```
Comment (schlessera) [inline src/Shell_Command.php]: Looks like a trailing space here.

**Comment by hearvox** [comment]
@schlessera, if you have time, code you describe the process for making a requested change on a pull requested. I see this not added by Github:
> Add more commits by pushing to the master branch on hearvox/shell-command.

So I'm guessing, rather than me closing this PR and making a new PR with requested, there's a way to make the change in my forked repo and merge it with this existing PR. But I can't find the GitHub doc on how to do that.

I made the change in my forked repo (removed trailing space on hearvox/shell-command). How do I get that change into this PR? thanks

**Comment by schlessera** [comment]
@hearvox The change was indeed added to this PR (look at the entry above your last comment).

In general, though, I'd recommend always creating a branch for doing a PR. This allows you to have more than one PR, and you can switch between them and add to them as needed.

**Comment by schlessera** [comment]
Authoring props: @hearvox & @ako80218

**Pull request merged**